docs/80_hardware/arch.md

```diff
@@ -0,0 +1,24 @@
+# Processor Chipset Architecture
+<div style="text-align: center;">
+  <img src="./SoC.png" alt="Placeholder Image" style="width:50%;">
+  <p><em>Intel Meteror Lake processor architecture. Modern day IC vendors tends to integrate various computation components on one single chipset to facilitate performance. When performing bioimage analysis we often utilise the processor's different computational units. Certain processor architectures facilitate more one specific tasks, e.g. image decode/encode tasks can take advantage of Intel Integrated Performance Primitives (IPP) library with hardware level accelerations.</em></p>
+</div>
+
+Modern days computer CPUs are more lean to a System-on-a-Chip (SoC) that integrates all major components of a computing device including CPU, GPU, NPU and RAM. The physically compactness brings shorter communication route among each computing units, hence facilitate computing performance.
```

Comment:
Do you think that detailed knowledge of what SoC are is relevant for the target audience (Python beginners who strive to analyse images)?

Reply:
I will remove all technical terms.